bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -228,6 +228,14 @@ This certificate is used for pairing between the Bridge and the Bosch Smart Home
 
 _Press and hold the Bosch Smart Home Controller Bridge button until the LED starts blinking after you save your settings for pairing_.
 
+### Supported Channels
+
+| Channel ID         | Item Type | Writable | Description                                                             |
+|--------------------|--------|:--------:|-------------------------------------------------------------------------|
+| scenario-triggered | String | &#9744;  | Name of the triggered scenario (e.g. by the Universal Switch Flex)      | 
+| trigger-scenario   | String | &#9744;  | Scenario name that will be triggered on the Bosch Smart Home Controller |
```

Comment:
1. Proposal: `Name of a scenario to be triggered on the Bosch Smart Home Controller.`
2. The `trigger-scenario` channel is writable, so the icon is wrong: it should be `&#9745;` instead of `&#9744;`